app/helpers/recommendations_helper.rb

```diff
@@ -66,26 +66,32 @@ def recommendations_sites
     {
       "stop-the-tories" => {
         order: 1,
+        name: "Stop The Tories",
         link: "https://stopthetories.vote/"
       },
       "sprint-for-pr" => {
         order: 2,
+        name: "Sprint for PR",
         link: "https://sprintforpr.uk/plan"
       },
       "tactical-vote" => {
         order: 3,
+        name: "Tactical (dot) Vote",
         link: "https://tactical.vote/all/"
```

Comment:
Was it deliberate to have the `all/` path here?  Just curious, since all the other links go directly to the homepage.

Reply:
I was trying to direct people to a page that would answer the question "why was this recommended".  The line just above show something similar for sprintForPr.

Reply:
Fine by me, was just checking it was deliberate :+1: